semantic_segmentation/code/args.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+
+
+def get_args(desc="No Desc", data=False):
+    if not data:
+        parser = argparse.ArgumentParser(description=desc)

```

Comment:
we had to make the argument names full instead of just letters right?

Reply:
Yes

Reply:
We can have both `--` for full name and `-` for short